lib/WUI/wui_api.c

```diff
@@ -365,46 +358,15 @@ void add_time_to_timestamp(int32_t secs_to_add, struct tm *timestamp) {
     localtime_r(&current_time, timestamp);
 }
 
-uint16_t wui_upload_begin(const char *fname) {
-    // FIXME: We should start a new, independent download instead of using globals.
-    // Reset to a new upload.
-    upload_file = NULL;
-    const size_t fname_length = strlen(fname);
-
-    if ((fname_length + USB_MOUNT_POINT_LENGTH) < sizeof(tmp_filename)) {
-        strcpy(tmp_filename, USB_MOUNT_POINT);
-        strcpy(tmp_filename + USB_MOUNT_POINT_LENGTH, fname);
-        upload_file = fopen(tmp_filename, "w");
-        if (upload_file == NULL) {
-            // Missing USB -> Insufficient storage.
-            return 507;
-        } else {
-            return 0;
-        }
-    } else {
-        assert(0);
-        // Our own screwup, too long temp file name.
-        return 500;
-    }
+uint32_t wui_gcodes_uploaded() {
+    return uploaded_gcodes;
 }
 
-uint16_t wui_upload_data(const char *data, size_t length) {
-    assert(upload_file);
-    const size_t written = fwrite(data, sizeof(char), length, upload_file);
-    if (written < length) {
-        // Data won't fit into the flash drive -> Insufficient stogare.
-        fclose(upload_file);
-        upload_file = NULL;
-        remove(tmp_filename);
-        memset(tmp_filename, 0, sizeof(tmp_filename));
-        return 507;
-    } else {
-        return 0;
-    }
-}
+void wui_uploaded_gcode(const char *filename, bool start_print) {
+    uploaded_gcodes++;
 
-uint16_t wui_upload_finish(const char *old_filename, const char *new_filename, bool start) {
-    uint16_t error_code = 0;
+    (void)filename;
+    (void)start_print;
```

Comment:
```suggestion
void wui_uploaded_gcode(const char */*filename*/, bool /*start_print*/) {
    uploaded_gcodes++;
```

Reply:
Technically, that's not valid `C`. Not having a variable name is valid in C++, but C afaik doesn't allow it (but compilers sometimes accept it).

Reply:
`uint16_t wui_upload_finish(const char *, const char *, bool);` was used before in `wui_api.h`.